src/ controllers/trainee/Controller.ts

```diff
@@ -0,0 +1,62 @@
+import { Request, Response } from 'express';
+
+class TraineeController {
+    static instance;
+    static getInstance = () => {
+        if (TraineeController.instance) {
+            return TraineeController.instance;
+        }
+        TraineeController.instance = new TraineeController();
+        return TraineeController.instance;
+    }
+create = (req: Request, res: Response ) => {
+    console.log(':::::::Create Trainee:::::::');
+    res.send({
```

Comment:
1. do proper validation if not name then throw an error with 400 code and error message
2. similarly validate if not id saying id is required